server/views/certificate/data-vis.jade

```diff
@@ -1,5 +1,5 @@
 meta(name='viewport', content='width=device-width, initial-scale=1')
-link(rel='stylesheet', href='/bower_components/bootstrap/dist/css/bootstrap.min.css')
+link(rel='stylesheet', href='http://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css')
```

Comment:
We don't rely on cdn's for core libs as they are often blocked in different countries.